ui/src/router/index.js

```diff
@@ -40,7 +40,17 @@ router.beforeEach((to, from, next) => {
     } else {
       next();
     }
-  } else {
+  }
+  else if (to.path=="/login")
+  {
```

Comment:
In JavaScript, curly braces are usually written on the same line as the keyword, not on a new line. For example:
```
if (condition) {
  // do this
}
```